src/lib/atoms/Image/Image.tsx

```diff
@@ -8,25 +7,73 @@ import Tooltip from '../../molecules/Tooltip';
 import Checkbox from '../../molecules/Checkbox';
 
 // Styles
-import './index.scss';
+import './Image.scss';
 
-function Image({
+export interface IImageProps extends HTMLAttributes<HTMLDivElement> {
+    /**
+     * Image path to display
+     */
+    src: string;
+    /**
+     * Will add a border when set to "true"
+     */
+    withBorder: boolean;
+    /**
+     * The property will add a checkbox and an overlay over the image.
+     */
+    selectMode: boolean;
+    /**
+     * Customize checkbox(will be rendered only when "selectMode" is set to "true").
+     */
+    checkboxProps: {};
+    /**
+     * Customize image tag with this property.
+     */
+    imageProps: HTMLAttributes<HTMLImageElement>;
+    /**
+     * The property will render "actions" when "selectMode" is not setted to "true". Any valid React node
+     */
+    actions: ReactNode;
+    /**
+     * Will add a title to the top of Image atom. Any valid React node
+     */
+    title: string;
+    /**
+     * Title for 'Tooltip'.
+     */
+    tooltipTitle: string | ReactNode;
+    /**
+     * Addional className
+     */
+    className: string;
+    /**
+     * Image atom also can be included in "Form" organism. It can be validated as other "Form" elements
+     */
+    isValid: boolean;
+    /**
+     * Empty state text for component
+     */
+    emptyText: string;
+}
+
+const Image: FC<Partial<IImageProps>> = ({
```

Comment:
Please remove Partial and in storybook Image Docs you can see all types descriptions.
```suggestion
const Image: FC<IImageProps> = ({
```